torchtitan/config/job_config.py

```diff
@@ -865,6 +865,11 @@ class Experimental:
     needs to ensure that the path can be imported.
     """
 
+    # "aten" (default), "inductor", "none"
+    comms_bucket_reorder_strategy: str = "aten"
```

Comment:
this one might be worth landing in main.  it's purely inductor-specific and used by simple-fsdp as well.
cc @ruisizhang123 @fegin  @tianyu-l